src/Utils/JsLoader.php

```diff
@@ -17,7 +17,7 @@ class JsLoader
     public static $cdn = 'https://cdn.jsdelivr.net/gh/atk4/google-address';
 
     /** @var string Javascript file version. */
-    public static $version = '2.2.2';
+    public static $version = '4.0.0';
```

Comment:
remove it, no CDN, JS files are versioned together with atk4/ui files

Reply:
remove CDN and set the actual as default fallback: `/public/atk-google-maps.min.js` but that path is unusable in a real env, so I change it `/assets/atk-google-maps.min.js`